### ISSUE TITLE
After installation, OctoRelay is not working anymore

### ISSUE THREAD

**Issue by storbi81** [Royrdan/Octoprint-Smart-Filament-Sensor#5]
If you install the plugin, your current installation of OctoRelay will not work. Or if you plan to install OctoRelay later: no, this will not work, too.

I have installed OctoRelay already on my machine, with two configured buttons in the navbar. If I install now the Smart Filament Sensor plugin, my configured buttons are gone, instead of them, there are the four default symbols R1, R2, R3 and R4. and nothing is working. 

I think it's related to this issue: [https://github.com/Royrdan/Octoprint-Smart-Filament-Sensor/issues/3](url)

So at the moment, I live with that.... the power control with OctoRelay is more important for me than the Filament Sensor.

**Comment by Royrdan** [comment]
Hi storbi81,
Thanks for your feedback.
I am still getting familiar with the code but I will look into your issue and hopefully come back with a fix or for more information soon.
Thanks

**Comment by Royrdan** [comment]
Hi @storbi81 
I have removed the GPIO cleanup from the beginning of the file as this was obviously cleaning up all the other plugins GPIO setups. I beleive the previous maintainer used this to ensure there were no conflicts with the GPIO but I dont believe this is necessary. I have fixed this up and will be releasing a new version in a few days. Let me know if you still have the problem after upgrading.
Thanks
